packages/documentation/src/shared/link-design/link-design.component.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, nothing } from 'lit';
+import { html } from 'lit/static-html.js';
+import { customElement, property } from 'lit/decorators.js';
+import { getTitleFromPath } from '../../utils';
+import { ModuleExport } from '@storybook/types';
+
+interface DesignParameter {
+  type: 'figma';
+  url: string;
+}
+
+@customElement('link-design')
+export class LinkDesignComponent extends LitElement {
+  @property({ type: Object }) of: ModuleExport = {};
```

Comment:
ModuleExport (which is used in `<Meta />` is equal to any at the moment, but it might change